.github/workflows/php-security.yml

```diff
@@ -5,16 +5,15 @@ on:
   secrets:
       SNYK_TOKEN:
         required: true
-  inputs:
-    upload-sarif:
-      required: false
-      type: boolean
-      default: false
+  
 jobs:
  snyk:
    runs-on: ubuntu-latest
    name: Snyk PHP Scan
-
+   permissions:
+      security-events: write
+      actions: read
+      contents: read
    steps:
     - name: Checkout code
       uses: actions/checkout@v3
```

Comment:
`php-static` and `php-tests` use `actions/checkout@v2` which causes a deprecated warning on the workflow output. Could they be bumped to `actions/checkout@v3` as part of this PR?

Reply:
yes good catch